File: api-examples/rs/count/js/temporal_after.js
var settings = {
  "async": true,
  "crossDomain": true,
  "url": "https://pune.iudx.org.in/resource-server/pscdcl/v1/count",
  "method": "POST",
  "headers": {
    "Content-Type": "application/json",
  },
  "processData": false,
  "data": "{\n\t\"id\": \"rbccps.org/aa9d66a000d94a78895de8d4c0b3a67f3450e531/pscdcl/aqm-bosch-climo/ABC Farm House Junction_4\",\n\t\"time\": \"2019-08-19T00:00:00.000Z\",\n\t\"TRelation\": \"after\"\n}"
}

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: api-examples/rs/count/java/temporal_after.java
OkHttpClient client = new OkHttpClient().newBuilder()
  .build();
MediaType mediaType = MediaType.parse("application/json");
RequestBody body = RequestBody.create(mediaType, "{\n\t\"id\": \"rbccps.org/aa9d66a000d94a78895de8d4c0b3a67f3450e531/pscdcl/changebhai/crowd-sourced-images\",\n\t\"time\": \"2019-09-26T06:18:54.717Z\",\n\t\"TRelation\": \"after\"\n}");
Request request = new Request.Builder()
  .url("https://localhost/resource-server/pscdcl/v1/count")
  .method("POST", body)
  .addHeader("Content-Type", "application/json")
  .build();
Response response = client.newCall(request).execute();
